Pythonwin/pywin/Demos/ocx/msoffice.py

```diff
@@ -123,6 +119,18 @@ def __init__(self, doc=None):
         # Dont call base class doc/view version...
 
     def Create(self, title, rect=None, parent=None):
+        WordModule = gencache.EnsureModule(
+            "{00020905-0000-0000-C000-000000000046}", 1033, 8, 0
+        )
+        if WordModule is None:
+            raise ImportError(
+                "Microsoft Word version 8 does not appear to be installed."
+            )
+
+        # WordModule.Word doesn't exist in WordModule, WordModule.Words does, but CreateControl still fails
```

Comment:
We could either leave this as-is if we'd like to fix it later (the calling code is still commented out, at least now tooling won't report undefined names / unbound variables).

Or delete the `WordFrame` class entirely (doesn't look like this part of the demo was ever run in the git history)